index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Promise: Portfolio</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+    integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <!-- first-section -->
+  <div class="wrapper">
+    <div class="mobile-view">
+      <div class="header-wrapper">
+        <!-- navbar-section -->
+        <header class="header-section">
+          <span class="nav-span">Promise Okechukwu</span>
+          <span><a href="#"><img src="./images/Enabled.png" class="#" alt="#"></a></span>
+         
+          <article class="nav-links">
```

Comment:
- [ ] It is important to know that you should use the proper tags in their proper role. Here you wrapped all your nav links into an `article` tag. This is not the article or any specific content so that means it was not the best place to use, rather I would suggest you wrap your nav links into **unordered list** By doing this, they will be represented well and also be easily accessible to search engines and accessibility devices 👍